README.rst

```diff
@@ -205,8 +253,8 @@ variables. Definition via environment variable is helpful in environments where
 it is not convenient to create a `.pypirc` file, such as a CI/build server, for
 example.
 
-* ``TWINE_USERNAME`` - the username to use for authentication to the repository
-* ``TWINE_PASSWORD`` - the password to use for authentication to the repository
+* ``TWINE_USERNAME`` - the username to use for authentication to the repository.
+* ``TWINE_PASSWORD`` - the password to use for authentication to the repository.
 * ``TWINE_REPOSITORY`` - the repository configuration, either defined as a
```

Comment:
For parallelism, could you add the ending period to all the items in this list?

Reply:
Done.